src/app/documentation/documentation.component.html

```diff
@@ -11,16 +11,20 @@ <h1 class="page-header">Documentation</h1>
                 <div class="field-item">
                   <h2>PDF Documentation</h2>
                   <p>PDF documentation can be downloaded from the following links.</p>
+                  <p>
+                    Note that starting with 25.1.0, some documentation is being hosted on <a href="https://energyplus.readthedocs.io/en/stable/index.html">ReadTheDocs</a> instead of provided as PDFs.
+                    For now this includes the Acknowledgments, Auxiliary Programs, EMS Application Guide, and EnergyPlus Essentials.
+                  </p>
                   <p>
                     <a class="btn btn-primary btn-lg btn-bg-dark" role="button" href="assets/nrel_custom/pdfs/pdfs_v{{latestRelease.version}}/pdfs.zip" download>Download All PDFs</a>
                   </p>
                   <div class="row">
                     <div class="col-xs-12 col-sm-4">
                       <div class="btn-group-vertical btn-block" role="group">
-                        <a href="assets/nrel_custom/pdfs/pdfs_v{{latestRelease.version}}/Acknowledgments.pdf" class="btn btn-body border text-left" type="button">Acknowledgements</a>
-                        <a href="assets/nrel_custom/pdfs/pdfs_v{{latestRelease.version}}/AuxiliaryPrograms.pdf" class="btn btn-body border text-left" type="button">Auxiliary Programs</a>
-                        <a href="assets/nrel_custom/pdfs/pdfs_v{{latestRelease.version}}/EMSApplicationGuide.pdf" class="btn btn-body border text-left" type="button">EMS Application Guide</a>
-                        <a href="assets/nrel_custom/pdfs/pdfs_v{{latestRelease.version}}/EnergyPlusEssentials.pdf" class="btn btn-body border text-left" type="button">EnergyPlus Essentials</a>
+                        <a href="https://energyplus.readthedocs.io/en/stable/acknowledgments/acknowledgments.html" class="btn btn-body border text-left" type="button">Acknowledgements</a>
+                        <a href="https://energyplus.readthedocs.io/en/stable/auxiliary-programs/auxiliary-programs.html" class="btn btn-body border text-left" type="button">Auxiliary Programs</a>
+                        <a href="https://energyplus.readthedocs.io/en/stable/ems-application-guide/ems-application-guide.html" class="btn btn-body border text-left" type="button">EMS Application Guide</a>
+                        <a href="https://energyplus.readthedocs.io/en/stable/essentials/essentials.html" class="btn btn-body border text-left" type="button">EnergyPlus Essentials</a>
```

Comment:
I think to a user this should be pretty smooth to just go to RTD instead of downloading a PDF.